api/inventory/v1/agents.proto

```diff
@@ -1181,6 +1224,54 @@ message ChangeQANMongoDBProfilerAgentParams {
   common.MetricsResolutions metrics_resolutions = 4;
 }
 
+// Add/Change QANMongoDBMongologAgent
+
+message AddQANMongoDBMongologAgentParams {
+  // The pmm-agent identifier which runs this instance.
+  string pmm_agent_id = 1 [(validate.rules).string.min_len = 1];
+  // Service identifier.
+  string service_id = 2 [(validate.rules).string.min_len = 1];
```

Comment:
```suggestion
  string service_id = 2 [(validate.rules).string.uuid = true];
```

Reply:
As Michael point out we cannot use this due to "pmm-server".